app/js/lib/foxbox.js

```diff
@@ -425,7 +453,7 @@ export default class Foxbox extends Service {
               .then((state) => service.state = state);
           })
         ).then(() => services);
-      });
+      });*/
```

Comment:
I'm not completely sure, but you're right we can get rid of it and return it back once/if we need it.